apps/server/src/modules/news/controller/dto/user-info-response.dto.ts

```diff
@@ -0,0 +1,12 @@
+import { Expose } from 'class-transformer';
+
+export class UserInfoResponseDto {
```

Comment:
Can this extend like the user entity or somewhat to ensure the given properties are in the user or do we prefer not couple it to the user?

Reply:
i would say we should not couple dtos to entities tightly. dtos can expose subsets or combinations of entities, eg. aggregates or compositions.